test/tools/OpenCover/OpenCover.psm1

```diff
@@ -412,8 +412,8 @@ function Invoke-OpenCover
     }
 
     $targetArgString = $targetArgs -join " "
-    # the order seems to be important
-    $openCoverArgs = "-target:$target","-targetargs:""$targetArgString""","-register:user","-output:${outputLog}","-nodefaultfilters","-oldstyle","-hideskipped:all"
+    # the order seems to be important. Always keep -targetargs as the last parameter.
+    $openCoverArgs = "-target:$target","-register:user","-output:${outputLog}","-nodefaultfilters","-oldstyle","-hideskipped:all","-targetargs:""$targetArgString"""
```

Comment:
I means using `" to represent a literal double quote character within a double quoted string. It will still be a double quote when the string is evaluated.